api/views/purchase.py

```diff
@@ -103,8 +103,8 @@ class PurchaseListCreateView(ListCreateAPIView):
         "date",
         "provider",
         "price_ht",
-        "canteen__name",
```

Comment:
Won't this prevent the list being sortable by canteen name?

Reply:
Thanks ! Fix:  https://github.com/betagouv/ma-cantine/pull/2463/commits/7bd2cdb45653fbf4e7342c958045089a078c57af